src/main/java/com/wuest/from_the_depths/integration/SSHelper.java

```diff
@@ -0,0 +1,97 @@
+ * @author Davoleo
+ */
+public class SSHelper {
+
+    public static final BooleanSupplier isSereneSeasonLoaded = () -> Loader.isModLoaded("sereneseasons");
+    public static Map<String, Pair<Season, Season.SubSeason>> seasonRestrictions = new HashMap<>();
+
+    private static final BiPredicate<Pair<Season, Season.SubSeason>, World> SEASON_SPAWN_RESTRICTION = (season, world) -> {
+        ISeasonState worldSeason = SeasonHelper.getSeasonState(world);
+        if (season.getRight() == null)
+            return worldSeason.getSeason() == season.getLeft();
+        else
+            return worldSeason.getSeason() == season.getLeft() && worldSeason.getSubSeason() == season.getRight();
+    };
+
+    /**
+     * Parses {@link Season} and {@link sereneseasons.api.season.Season.SubSeason} objects from string parameters and returns a pair of them
+     * @param seasonStr Season string representation (should be the same name as {@link Season} constants)
+     * @param subSeasonStr SubSeason string representation (should be the same name as {@link sereneseasons.api.season.Season.SubSeason} constants)
+     * @throws IllegalArgumentException When any of the string representation of enum constants are wrong (the exception IS NOT handled here)
+     */
+    private static Pair<Season, Season.SubSeason> seasonFromStrings(String seasonStr, String subSeasonStr) throws IllegalArgumentException {
+
+        Season season = Season.valueOf(seasonStr);
+
+        Season.SubSeason subSeason = null;
+        if (!subSeasonStr.equals("")) {
+            subSeason = Season.SubSeason.valueOf(subSeasonStr + '_' + seasonStr);
+        }
+
+        return Pair.of(season, subSeason);
+    }
+
+    /**
+     * Adds a season restriction in the form of a Pair of seasons to the {@link SSHelper#seasonRestrictions} map
+     * @param spawnKey The boss key this season restriction is bound to.
+     * @param seasonObj The season restriction information (directly from the JSON)
+     * @throws JsonSyntaxException when the "season" attribute is not present
+     *      (Note that subSeason is optional and fallbacks to "" in case it's not given any value)
+     */
+    public static void addSeasonRestriction(String spawnKey, JsonObject seasonObj) throws JsonSyntaxException
+    {
+        String sString = JsonUtils.getString(seasonObj, "season");
+        String sSString = JsonUtils.getString(seasonObj, "subSeason", "");
+        Pair<Season, Season.SubSeason> seasons = seasonFromStrings(sString.toUpperCase(), sSString.toUpperCase());
+        seasonRestrictions.put(spawnKey, seasons);
+    }
+
+    /**
+     * @param world ZA WARUDO
+     * @return whether season restrictions are asserted for the passed boss spawn key.
+     */
+    public static Pair<Boolean, String> testSeasonRestrictions(String spawnKey, World world) {
+        //Return true if Serene Seasons is not installed
+        if (!isSereneSeasonLoaded.getAsBoolean())
```

Comment:
I'll put this here since it's a small file but something I noticed throughout the added code of the PR.

Please put brackets around the body of an "if", "else" or other similar block of code.

This makes it easier moving forward if more processing needs to be added to that code block in the future.

It also makes the code consistent and therefore easier to read.

Reply:
alright, yeah sorry about this, I did try to keep consistency when I noticed the project was using braces for if statements even when they only have 1 line body, although I have missed a few due to habits, I'll refactor those I see